client/src/components/Ticketing/ticketingmanager/Season/components/SeasonInfo.tsx

```diff
@@ -213,13 +214,12 @@ const SeasonInfo = (props: SeasonProps) => {
             />
           </label>
           <div id='form-checkboxes' className='flex gap-7'>
-            <div className='checkbox-2'>
-              <input
-                type='checkbox'
-                id='defaultImage'
-                name='defaultImage'
-                className='mr-2'
+            <div className='checkboxes'>
```

Comment:
I don't think this class does anything, I would instead maybe make this a flex object and do styling on this element rather using `sx` in the innner elements. On mobile, the checkboxes are a little weird

Reply:
![image](https://github.com/WonderTix/WonderTix/assets/92124260/1318c15d-0c95-4f90-9381-8820e971043d)

Reply:
I'm actually going to revert this back to the old checkbox and make the change in the next PR. I was working on the seasons active button but had to update the api being used for it to work properly and forgot to stash my changes.

Reply:
Sounds good 👍🏻